code/modules/mob/living/simple_animal/bot/bot_construction.dm

```diff
@@ -30,9 +30,11 @@
 
 	else if(is_pen(W))
 		var/t = rename_interactive(user, W, prompt = "Enter new robot name")
-		if(!isnull(t))
+		if(length(t) > 0)
```

Comment:
```suggestion
		if(length(t))
```
You can do it like this, its checking if the "t" has anything inside of it, which would mean it has more than 0 already.